Assets/StreamingAssets/Quests/R0C11Y28.txt

```diff
@@ -364,12 +364,12 @@ Message:  1050
 --              %god occurs 3 times.
 --               %mn occurs 3 times.
 --              %pcf occurs 11 times.
---              %pcn occurs 2 times.
+--              %___bookstore_ occurs 2 times.
 --             %qdat occurs 6 times.
 --               %ra occurs 1 time.
 --              %reg occurs 10 times.
 --               %rt occurs 1 time.
---              %vcn occurs 3 times.
+--              %___bookstore_ occurs 3 times.
```

Comment:
Same here, %vcn.

Reply:
Wow, that's terrible. Thanks a lot for proofreading that. I don't know how find and replace can go _that_ badly, but it did!